--- nacl_sdk/sdk_tools/download.py	2018-07-01 17:56:50.039963802 +0000
+++ nacl_sdk/sdk_tools/download.py	2018-07-01 17:56:50.039963802 +0000
@@ -16,8 +16,8 @@

 def UrlOpen(url):
   request = fancy_urllib.FancyRequest(url)
-  ca_certs = os.path.join(SCRIPT_DIR, 'cacerts.txt')
-  request.set_ssl_info(ca_certs=ca_certs)
+  #ca_certs = os.path.join(SCRIPT_DIR, 'cacerts.txt')
+  #request.set_ssl_info(ca_certs=ca_certs)
   url_opener = urllib2.build_opener(
       fancy_urllib.FancyProxyHandler(),
       fancy_urllib.FancyRedirectHandler(),
